tests/integration/frameworks/test_agent.py

```diff
@@ -113,6 +113,7 @@ class Steps(BaseModel):
     [
         "anthropic/claude-3-5-haiku-latest",
         "gemini/gemini-2.5-flash",
+        "huggingface/tgi",  # This is a Qwen/Qwen3-1.7B hosted in https://endpoints.huggingface.co/mozilla-ai/endpoints/dedicated
```

Comment:
Is there some way to make this a specific endpoint name? My worry is that we have no way of knowing if it is no longer Qwen3-1.7B